src/main/java/tsunami/security/scanner/utilities/App.java

```diff
@@ -16,12 +16,180 @@
+        String kubeConfigPath = System.getProperty("user.home") + "/.kube/config";
+
+        // loading the out-of-cluster config, a kubeconfig from file-system
+        ApiClient client =
+                ClientBuilder.kubeconfig(KubeConfig.loadKubeConfig(new FileReader(kubeConfigPath))).build();
+
+        // set the global default api-client to the in-cluster one from above
+        Configuration.setDefaultApiClient(client);
+
+
+        // TODO: load kustomization or secret file (unfixed problem in creating namespaced secret object)
+        /*
+        String secretPath = System.getProperty("user.dir") + "/application/wordpress/mysql-pass.yaml";
+        File secretFile = new File(secretPath);
+        V1Secret secret = (V1Secret) Yaml.load(secretFile);
+        System.out.println(secret);
+        */
+
+        /* Initialize apis */
+        CoreV1Api api = new CoreV1Api();
+        AppsV1Api api1 = new AppsV1Api();
+
+        /* Load mysql config file */
+        String mysqlConfigPath = System.getProperty("user.dir") + "/application/wordpress/mysql-deployment.yaml";
+        File mysqlFile = new File(mysqlConfigPath);
+        List<Object> objectList4Mysql = (List<Object>) Yaml.loadAll(mysqlFile);
+
+        /* Load wordpress config file */
+        String wordpressConfigPath = System.getProperty("user.dir") + "/application/wordpress/wordpress-deployment.yaml";
+        File wordpressConfig = new File(wordpressConfigPath);
+        List<Object> objectList4Wordpress = (List<Object>) Yaml.loadAll(wordpressConfig);
+
+        /* Parse mysql configs */
+        V1Service mysqlSvc = (V1Service) objectList4Mysql.get(0);
+        V1PersistentVolumeClaim mysqlPvc = (V1PersistentVolumeClaim) objectList4Mysql.get(1);
+        V1Deployment mysqlDeployment = (V1Deployment) objectList4Mysql.get(2);
+
+        /* Parse wordpress configs */
+        V1Service wordpressSVC = (V1Service) objectList4Wordpress.get(0);
+        V1PersistentVolumeClaim wordpressPVC = (V1PersistentVolumeClaim) objectList4Wordpress.get(1);
+        V1Deployment wordpressDeployment = (V1Deployment) objectList4Wordpress.get(2);
+
+
+        /* Create services */
+        V1Service createResultSvc = api.createNamespacedService("default", mysqlSvc, null, null, null);
+        V1Service createResultWordpressSvc = api.createNamespacedService("default", wordpressSVC, null, null, null);
+
+        /* Create persistent volume claims */
+        V1PersistentVolumeClaim createResultPvc= api.createNamespacedPersistentVolumeClaim("default", mysqlPvc, null,null,null);
+        V1PersistentVolumeClaim createResultWordpressPvc= api.createNamespacedPersistentVolumeClaim("default", wordpressPVC, null,null,null);
+
+        /* Deploy */
+        V1Deployment createResultDeployment= api1.createNamespacedDeployment("default", mysqlDeployment, null,null,null);
+        V1Deployment createResultWordpressDeployment= api1.createNamespacedDeployment("default", wordpressDeployment, null,null,null);/*/
```

Comment:
You can simplify this code a lot by using a map from `Object.getClass()` to the API call. You can find a similar solution in [this github issue](https://github.com/kubernetes-client/java/issues/965).

Reply:
Let's resolve this in a follow up PR.